src/utils/BucketUtils.js

```diff
@@ -0,0 +1,251 @@
+import {any, compact, findIndex, filter, flatMap, flow, forEach, get, getOr, includes, isEmpty, isEqual, isUndefined, join, map, toLower, uniq, values} from 'lodash/fp';
```

Comment:
Part of the problem is that the buckets are different depending on the view. As an admin, buckets are unfiltered. As a chair, buckets have non-dac datasets/votes/elections filtered out. This is purely a UI preference since there is no systemic reason why a dac chair cannot know about elections in other DACs. I can see the appeal of moving it to the back end, but we're going to change our minds on a somewhat regular basis about what a view/voting screen looks like. Having a stable backend API that provides the correct data will let us be flexible in how we display it.